lib/maths/time_series/CTrendComponent.cc

```diff
@@ -52,6 +52,24 @@ const core_t::TTime UNSET_TIME{0};
 const std::size_t NO_CHANGE_LABEL{0};
 const std::size_t LEVEL_CHANGE_LABEL{1};
 
+class CChangeForecastFeatureWeight : public common::CNaiveBayesFeatureWeight {
+public:
+    void add(std::size_t class_, double logLikelihood) override {
+        if (class_ == NO_CHANGE_LABEL) {
+            m_LogLikelihood = logLikelihood;
+        }
+    }
+
+    double calculate() const override {
+        // Downweight features for which we don't have sufficient examples
+        // of the time series not changing.
+        return common::CTools::logisticFunction((4.5 + m_LogLikelihood) / 4.5, 0.1);
```

Comment:
Maybe, you can add some more information abou the implementation decision here: where does 4.5 come from?

Reply:
I added an explanation of the fact that 4.5 = 3 sigma point of the distribution.